core/config/locales/en.yml

```diff
@@ -51,11 +51,12 @@ en:
         name: Add File
         current: Current File
       search:
-        button_text: Search
+        button_text: Go
```

Comment:
How about "Search for..." or "Search terms"  (expected audience experienced enough to understand 'search terms').

Translations: Given that we are not changing the meaning of the field, just providing a more fluent and less verbose English translation, what do we need to do wrt other languages.  I had a look at `it.yml` and they do seem to have translated word for word.
Do we alert them to changed fields? (which them?) 

Also, I would anticipate that in the nearish future, I would like to re-present this field, with a placeholder in the field and a search icon on the button. (and the parts connected smoothly to each other).

I have to go now, but will perhaps start a conversation about backend presentation in a better spot than this.